controllers/errors.js

```diff
@@ -1,3 +1,16 @@
 exports.invalidPath = (req, res) => {
   res.status(404).send({ msg: "Path not found!" });
 };
+
+exports.handleCustomErrors = (err, req, res, next) => {
+  console.log("inside handleCustomErrors");
```

Comment:
Remove console logs once the code is completed and you are no longer trying to debug

Reply:
Done